### DISCUSSION TITLE
Upgrading from 19.4.x to 20: recurring data is not saved

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#3908] [Questions & Answers]
Data entered in the recurring section of a product does not seem to be saved ( or loaded in the gui). Recurring products created in 19.4.x also have this data now shown. However cron jobs still create the periodical orders.

![recurring](https://github.com/OpenMage/magento-lts/assets/13450219/dd93f485-6453-4e5d-8d65-fe1fb35af3c8)
after saving, this form is empty

**Comment by f1-outsourcing** [comment]
If I switch back to magento 19.4.x, I see the data in the recurring profile I saved under 20.5.x. So I guess it is just not loading and showing it. (and maybe erasing after pressing a 2nd time save)

**Comment by fballiano** [comment]
I can confirm this seems to be a bug

**Reply by fballiano** [reply]
it was working in 20.2.0 but broke in 20.3.0

**Reply by addison74** [reply]
I will create an issue from this discussion.

**Reply by fballiano** [reply]
from a good run with git bisect it seems https://github.com/OpenMage/magento-lts/pull/3663 is the cause of the bug

**Reply by fballiano** [reply]
I can confirm that #3663 is the root of this bug
